builder/softlayer/step_capture_image.go

```diff
@@ -40,6 +86,33 @@ func (self *stepCaptureImage) Run(state multistep.StateBag) multistep.StepAction
 		return multistep.ActionHalt
 	}
 
+	if config.ImageType == "standard" {
```

Comment:
In continue to my previous [comment](https://github.com/leonidlm/packer-builder-softlayer/pull/12/files#r22442596) can this be done before blocking for image creation to finish?

Also, maybe we should extract the if block into a separate function (`getImageIdByName` or something...)? I think it will make the code cleaner.